features/manage_meeting_rooms.feature

```diff
@@ -4,11 +4,23 @@ Feature: Manage Meeting Rooms
   I want to create and delete meeting rooms
 
   Scenario: Create Meeting Room
```

Comment:
Please be more descriptive of the scenario being tested. I'd say this would be something like this `A user can create a Meeting room by sending a POST request with all required parameters`

Reply:
Sure will change with my next commit.